bigquery/deploy/README.md

```diff
@@ -40,7 +12,12 @@
     gcloud container clusters get-credentials elasticsearch-cluster --zone MY_ZONE
     ```
     This will make `kubectl` use this cluster.
-  * Run [kubectl commands](https://github.com/pires/kubernetes-elasticsearch-cluster#deploy)
+
+* Run Elasticsearch on GKE
+  * Deploy Elasticsearch:
+    ```
+    ./bigquery/deploy/kubernetes-elasticsearch-cluster/deploy.sh
```

Comment:
I think you have to cd first, otherwise there's an error

Reply:
Done